README.md

```diff
@@ -40,10 +40,12 @@ which supports SystemVerilog and UVM 1.2. This generator has been verified with
 Synopsys VCS, Cadence Incisive/Xcelium, and Mentor Questa simulators. Please
 make sure the EDA tool environment is properly setup before running the generator.
 
-Install dependencies needed for running:
+Install RISCV-DV:
 
 ```bash
-pip3 install -r requirements.txt
+git clone https://github.com/google/riscv-dv.git
+cd riscv-dv
+pip3 install -e .
```

Comment:
Can you add some instruction about how to update the installed package to the upstream version? Since we are changing the script very often, I hope user can quickly sync and run with the latest version.

Reply:
Yes. I think about it. I believe user can get all everything by only one command:
`pip3 install --user git+https://github.com/google/riscv-dv.git`. But I cannot test it until these commit are merged.
I can locally test by : `pip3 install -e .`. The output:
```
Obtaining file:///home/danghai/hhoangda/riscv-dv
Collecting PyYAML (from riscv-dv==1.0)
Collecting bitstring (from riscv-dv==1.0)
  Using cached https://files.pythonhosted.org/packages/c7/b9/ce7458b35633968eae3477a201a00f6c8e01cab0ccb200ea5099d41dd977/bitstring-3.1.6-py3-none-any.whl
Installing collected packages: PyYAML, bitstring, riscv-dv
  Found existing installation: riscv-dv 1.0
    Uninstalling riscv-dv-1.0:
      Successfully uninstalled riscv-dv-1.0
  Running setup.py develop for riscv-dv
Successfully installed PyYAML-5.2 bitstring-3.1.6 riscv-dv
```
What about merging these things first, and then I can update for next commit?

Reply:
Let me add `MANIFEST` file for Including files in source distributions